client/app/bundles/course/assessment/components/InfoLabel/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { CSSProperties, ReactNode } from 'react';
+import { Typography, Box } from '@mui/material';
+import {
+  WarningAmber as WarningIcon,
+  InfoOutlined as InfoIcon,
+} from '@mui/icons-material';
+
+interface InfoLabelProps {
+  label?: string;
+  children?: ReactNode;
+  warning?: boolean;
+  mt?: number;
```

Comment:
Originally I wanted to save horizontal space by using `mt` so that for a simple information label, the code need not break like such:
```jsx
<InfoLabel
  label={translations.someTranslations}
  marginTop={1}
/>
```

But then after seeing that even with `mt` it breaks like that, I have since changed it back to `marginTop`.